### DISCUSSION TITLE
Swagger Schema?

### DISCUSSION THREAD

**Discussion by dmayle** [ory/kratos#919] [Q&A] [answered]
The docs say that they are generated from the Swagger schema, which is in-turn generated from the code.

Is the schema available anywhere for download?  If not, are there instructions on how to generate it from the code?

**Comment by vinckr** [comment]
Unfortunately we do not have extensive documentation on how our documentation is built but I think you are looking for our [documentation-meta repository](https://github.com/ory/docs).
There you can find [Swagger / OpenAPI 2 and OpenAPI 3 templates](https://github.com/ory/docs/tree/master/.widdershins/templates).
Is that what you were looking for?

**Comment by zepatrik** [comment, marked as answer]
The Kratos schema is here: https://github.com/ory/kratos/blob/master/.schema/api.swagger.json

**Comment by SkypLabs** [comment]
Quick question related to this discussion (I don't think it's worth creation a new discussion for it): why is the [OpenAPI definition available on your website](https://www.ory.sh/docs/kratos/reference/api) different from the [one of the latest stable version present in this Git repository](https://raw.githubusercontent.com/ory/kratos/v0.11.0/.schema/openapi.json)?
